src/smc-webapp/editor_chat.cjsx

```diff
@@ -431,6 +443,17 @@ Message = rclass
             </div>
         </Col>
 
+    is_user_editing: ->
+        other_editors = @props.message.get('editing').remove(@props.account_id).keySeq()
+        current_user = @props.user_map.get(@props.account_id).get('first_name') + ' ' + @props.user_map.get(@props.account_id).get('last_name')
```

Comment:
This is a potential weird definition of current_user... which seems to not even be used at all in this function?